imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -365,13 +365,16 @@ def populate_full_cycle_data(
     calibrated_counts = apply_in_flight_calibration(full_cycle_data, acquisition_times)
 
     # Convert counts to rate
-    counts_rate = convert_counts_to_rate(calibrated_counts, acq_duration)
+    counts_rate = convert_counts_to_rate(
+        calibrated_counts, acq_duration_arr[:, :, np.newaxis]
+    )
 
-    # Store count data and acquisition times of full cycle data in xr.Dataset
+    # Store these data of full cycle data in xr.Dataset for later use.
```

Comment:
```suggestion
    # Store full cycle data in xr.Dataset for later use.
```

Suggestion for clarity.